stl/inc/__msvc_bit_utils.hpp

```diff
@@ -140,9 +140,26 @@ _NODISCARD int _Checked_x86_x64_countl_zero(const _Ty _Val) noexcept {
 
 #if defined(_M_ARM) || defined(_M_ARM64)
 #ifdef __clang__ // TRANSITION, GH-1586
+#ifdef _M_ARM64
+_NODISCARD constexpr int _Clang_arm_arm64_countl_zero(const unsigned char _Val) {
+    // TODO: Revert back to __builtin_clzs once clang longer generates worse code than __builtin_clz

```

Comment:
After there's an upstream llvm-project issue that has been reported, these comments should say `// TRANSITION, LLVM-nnnn` citing the bug number. (We don't say TODO in the product code because IDEs pick that up and tell users that they have TODOs to take care of, which is confusing for them.)